### PR TITLE
Add view mode for paragraphs to index section pages

### PR DESCRIPTION
**Pull request by danquah**
Section pages could not really be indexed as they consists of nested paragraphs. This PR introduces a view mode for indexing and configures searchapi to render section-pages using this viewmode.

Also

DDSDK-385

### CODE REVIEW

File: web/themes/custom/mungo/mungo.theme [hunk 1]
```diff
@@ -93,16 +93,17 @@ function mungo_preprocess_field(&$variables) {
       $variables['item_col_classes'][] = 'col-xs-6';
     }
   }
-  if ($field_name == 'field_section') {
+  if ($field_name === 'field_section') {
     $prevkey = NULL;
+    $herobannerfound = false;
```
Comment (tripox) [inline web/themes/custom/mungo/mungo.theme]: `false` skal være `FALSE`

File: web/themes/custom/mungo/mungo.theme [hunk 2]
```diff
@@ -93,16 +93,17 @@ function mungo_preprocess_field(&$variables) {
       $variables['item_col_classes'][] = 'col-xs-6';
     }
   }
-  if ($field_name == 'field_section') {
+  if ($field_name === 'field_section') {
```
Comment (tripox) [inline web/themes/custom/mungo/mungo.theme]: 👍

**Comment by tripox** [comment]
Also want to make sure, you're aware that `composer.lock` changed and did it on purpose.

**Comment by danquah** [comment]
Yup, composer.lock was updated as a part of "Upgrade search_api and search_api_solr"

- codestyle has been fixed

**Comment by reload-branching-check** [comment]
**Reload Commit Checker**
The following commits does not correctly reference a Jira issue:

|SHA|Message|Author|Date|
|---|-------|------|----|
|1b92cbeb60a4e9156744366ffeb1f8ca303ef292|Removing exclude from search from default view-mode|Mads H. Danquah (mads@danquah.dk)|2018-02-20T19:33:56Z|

**Pull request merged**